glossary.md

```diff
@@ -70,6 +75,11 @@ Sushiswap. For instance LP bonds require SLP tokens of the OHM / DAI pair.
 Total Value Locked, is the dollar amount of all OHM staked in the protocol. This
 metric is often used as growth or health indicator in DeFi projects.
 
+### TaaS
+
+Treasury as a Service, is the business model of decentralized custody of
+partnership funds. OlympusDAO is designed for TaaS by selling bonds.
```

Comment:
```suggestion
partnership funds. OlympusDAO is designed for TaaS by selling bonds and absorbing partners' funds into its treasury as a result.
```

Reply:
I would change funds to "liquidity" @kschan6

Reply:
```suggestion
partnership funds. OlympusDAO is designed for TaaS by selling bonds and absorbing partners' liquidity into its treasury as a result.
```